src/components/publishing/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const sizeMeRefreshRate = 500
+
+export const zIndex = (size: string) => {
+  const sizes = {
+    modal: 1070,
+  }
+  return sizes[size]
+}
```

Comment:
Why is this a function rather than a single object?

Reply:
Ah yeah you're right this is overkill